src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -108,6 +108,7 @@ export class NotebookMarkdownRenderer {
 			return '<img ' + attributes.join(' ') + '>';
 		};
 		renderer.link = (href: string, title: string, text: string): string => {
+			let hrefAbsolute: boolean = path.isAbsolute(href);
```

Comment:
I'd add a comment here about why you're doing this at the top instead of down where it's used, assuming that the stuff we do between is the reason for why this is the case.